astropy/wcs/base_low_level_api.py

```diff
@@ -0,0 +1,248 @@
+import os
+import abc
+
+import numpy as np
+
+__all__ = ['BaseLowLevelWCS']
+
+
+class BaseLowLevelWCS(metaclass=abc.ABCMeta):
+    """
+    Abstract base class for the low-level WCS interface described in
+    *APE 14: A shared Python interface for World Coordinate Systems*
+    (https://doi.org/10.5281/zenodo.1188875)
+    """
+
+    @abc.abstractproperty
```

Comment:
`abstractproperty` has been deprecated since 3.3, it should be
```
@property
@abc.abstractmethod
```

Reply:
Fixed